### ISSUE TITLE
Rspec matcher deprecation warning

### ISSUE THREAD

**Issue by jerefrer** [rspec-given/rspec-given#12]
I just got this in a freshly created project:

```
--------------------------------------------------------------------------------
Capybara::RSpecMatchers::HaveText implements a legacy RSpec matcher
protocol. For the current protocol you should expose the failure messages
via the `failure_message` and `failure_message_when_negated` methods.
(Used from /home/jeremy/.rvm/gems/ruby-2.0.0-p643/gems/given_core-3.7.1/lib/given/rspec/monkey.rb:21:in `handle_matcher')
--------------------------------------------------------------------------------
```

Thanks a lot for the gem btw, I use it in every project ;)

**Comment by jkowens** [comment]
@jerefrer This is a Capybara issue that was fixed in version 2.3. See jnicklas/capybara/pull/1219.

**Comment by jerefrer** [comment]
Indeed I had capybara stuck at `~> 2.2.1` in my Gemfile ... Can't believe I didn't try that before posting. Thanks :+1:
